host/sgx/sgxquote.c

```diff
@@ -253,6 +251,44 @@ static bool _load_sgx_dcap_qvl(void)
     return (_qvl_module != NULL);
 }
 
+static bool _sgx_use_in_process_quoting()
+{
+    // Before: the in-process call path is used unless the environment variable
+    // SGX_AESM_ADDR is set. Now: the environment variable SGX_AESM_ADDR is
+    // deprecated. Out-of-process call path will be default on Linux as an SGX
+    // kernel patch upstreaming is scheduled in February or March 2021. Since
+    // the situation is not urgent on Windows, in-process call path will remain
+    // default on Windows.
+    bool result;
+    char* sgx_use_in_process_quoting = oe_dupenv("SGX_USE_IN_PROCESS_QUOTING");
+
+    if (sgx_use_in_process_quoting &&
```

Comment:
Need to cover the cases when "SGX_USE_IN_PROCESS_QUOTING" is set to other non-zero values. Also need to reset `defined_sgx_use_in_process_quoting` when "SGX_USE_IN_PROCESS_QUOTING" is not set.

```
    if (sgx_use_in_process_quoting == NULL)
    {
        defined_sgx_use_in_process_quoting = false;
#ifdef _WIN32
        result = true;
#else
        result = false;
#endif
    }
    else
    {
        defined_sgx_use_in_process_quoting = true;
        result = (strcmp(sgx_use_in_process_quoting, "0") != 0);
    }
```

Reply:
So if `sgx_use_in_process_quoting` has a value != 0, in-process quoting should be used, like the case where `sgx_use_in_process_quoting` = 1?